docs/static/hour-of-code/educators.html

```diff
@@ -0,0 +1,168 @@
+        <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,200;0,300;0,400;0,500;0,600;1,200;1,300;1,400;1,500;1,600&display=swap" rel="stylesheet">
+        <link rel="stylesheet" href="./styles.css" />
+
+        <script>
+            function handleOnLoad() {
+                document
+                    .querySelector('#openhome')
+                    .addEventListener('click', function () {
+                        window.pxtTickEvent('hourofcode.home');
+                    });
+
+                var langPicker = document.querySelector('#langpicker');
+                if (langPicker) {
+                    langPicker.remove();
+                }
+            }
+        </script>
+    </head>
+
+    <body id="root" class="root background-color-secondary" onload="handleOnLoad()">
+        <div id="main" role="presentation">
+            <nav>
+                <div class="arcade-logo">
+                    <a href="/">
+                        <img src="../logo.svg" alt="MakeCode Arcade" />
+                    </a>
+                </div>
+
+                <div class="microsoft-logo">
+                    <a
+                        href="https://www.microsoft.com/en-us/makecode"
+                        target="_blank"
+                    >
+                        <img src="../Micorsoft_logo_rgb_W-white_D.png" alt="Microsoft" />
+                    </a>
+                </div>
+            </nav>
+
+            <header class='background-color-primary'>
+                <div class="container">
+                    <div class="left-content-block">
+                        <h3 class='body-font'>Microsoft MakeCode Arcade</h3>
+                        <h1 class="display-font"><span class='first-line'>Hour</span><span class='small-heading'>of</span><span class='last-line'>Code<em class='trademark'>™</em></span></h1>
+                        <h4 class="display-font">2020</h4>
+                        <h5>Educator Information</h5>
+                    </div>
+
+                    <div class="right-content-block">
+                        <img src="./simulator.png" alt="MakeCode Arcade Simulator" />
+                    </div>
+                </div>
+            </header>
+
+            <section class='about container'>
+                <div class="content-block">
+                    <h2 class='display-font'>About MakeCode Arcade</h2>
+                    <div class="text-content-block">
+                        Microsoft MakeCode Arcade is a free, open source retro 2D 
+                        game development environment. Students anywhere from 9 to 18 
+                        can use MakeCode Arcade to design and create their own games 
+                        online that they can then share with friends and family or 
+                        download to hand-held game devices. Learn more about Microsoft 
+                        MakeCode at <a href='' target='_blank'>http://aka.ms/makecode</a>.
```

Comment:
```suggestion
                        MakeCode at <a href='https://makecode.com' target='_blank'>http://makecode.com</a>.
```

needs the href, but side question is there a benefit / need to refer to the aka.ms shortlink when we have a pretty / memorable url for the page always ?

Reply:
Added the href! The copy is from @Jaqster, so I'll direct the link question to her!

Reply:
The aka link actually directs to the About page (https://www.microsoft.com/en-us/makecode/about) - I use this for people who are brand new to MakeCode.  Does a better job explaining what the product is than the page full of code editor cards...